templates/share-modal.php

```diff
@@ -1,41 +1,60 @@
-<?php 
-    $url = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on' ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
+<?php
+/**
+ * Share
+ *
+ * Share modal template
+ *
+ * @package WordPress
+ * @subpackage community-portal
+ * @version 1.0.0
+ * @author  Playground Inc.
+ */
+?>
+<?php
+
+if ( ! empty( $_SERVER['REQUEST_URI'] ) && ! empty( $_SERVER['HTTP_HOST'] ) ) {
+	$request_uri = esc_url_raw( wp_unslash( $_SERVER['REQUEST_URI'] ) );
+	$http_host   = esc_url_raw( wp_unslash( $_SERVER['HTTP_HOST'] ) );
+
+	$url = isset( $_SERVER['HTTPS'] ) && 'on' === $_SERVER['HTTPS'] ? 'https' : 'http' . "://{$http_host}{$request_uri}";
```

Comment:
This line is adding a duplicate http host to url which is breaking the Facebook share.

Reply:
Push changed